source/view/videoController.js

```diff
@@ -15,63 +15,65 @@
 var videos = new Array();
 var videoID = 0;
 var playingVideo;
+var mediaManagerID;
+var videoManagerID;
 
-function loadVideo( src, type, dontRemoveWhenEnded ) {
+function loadVideo( src, type ) {
     var video = {
         "id" : videoID,
-        "elem" : document.createElement( "video" ),
-        "source" : document.createElement( "source" ),
-        "wrapper" : document.createElement( "div" )
     };
-    video.elem.appendChild( video.source );
-    video.wrapper.appendChild( video.elem );
-    video.source.src = "assets/video/" + src;
-    video.source.type = type || 'video/mp4;codecs="avc1.42E01E,mp4a.40.2"';
-    video.elem.id = "video" + video.id;
-    video.elem.className = "video";
-    video.wrapper.className = "videoWrapper";
-    video.elem.load();
+    video.source = "assets/video/" + src;
+    video.vidName = src;
 
     document.onkeypress = removeVideoOnEvent;
 
-    if ( !dontRemoveWhenEnded ) {
-        video.elem.onended = removeVideoOnEvent;
-    }
-
+    mediaManagerID = vwf_view.kernel.find( undefined, "/mediaManager" )[ 0 ];
+    videoManagerID = vwf_view.kernel.find( mediaManagerID, "videoManager" ) [ 0 ];
```

Comment:
It looks like the only time you're using `mediaManagerID` is on the next find. We can save ourselves a find by just doing `vwf_view.kernel.find( "", "//videoManager" )[ 0 ]`.